app/controllers/taggings_controller.rb

```diff
@@ -16,10 +16,12 @@ def destroy
     @tagging = current_user.taggings.find_by("tag_id = ? AND movie_id = ?", params[:tag_id], params[:movie_id])
     respond_to do |format|
       if @tagging && @tagging.destroy
-        format.js {}
+        format.turbo_stream {}
+        format.html { redirect_to movie_path(@movie), notice: 'Tag was destroyed.' }
```

Comment:
hmm, yeah, this is a weird one because it'll always be a turbo_stream response. but general handling of notices/errors can be part of the frontend redo.